src/Servers/Kestrel/test/FunctionalTests/HttpsTests.cs

```diff
@@ -402,43 +402,6 @@ public async Task HandshakeTimesOutAndIsLoggedAsDebug()
             Assert.Equal(LogLevel.Debug, loggerProvider.FilterLogger.LastLogLevel);
         }
 
-        [Fact]
-        public async Task ClientAttemptingToUseUnsupportedProtocolIsLoggedAsDebug()
```

Comment:
Had a lot of trouble w/ this test in previous builds. Was it known to be flaky in release/2.1❔ See [build #20221104.4](https://dev.azure.com/dnceng-public/public/_build/results?buildId=73348) for example.

On a similar note, latest build failed `AcceptsRequestWithChunkedEncodingAndUpgrade()` twice in two attempts. Should I remove that test too (or is it worthwhile to keep trying)❔
``` text
  Starting test group: Sockets.FunctionalTest/net461
  Microsoft (R) Test Execution Command Line Tool Version 15.9.0
  Copyright (c) Microsoft Corporation.  All rights reserved.
  
  Starting test execution, please wait...
  [xUnit.net 00:00:02.40]     AcceptsRequestWithChunkedEncodingAndUpgrade [FAIL]
  Failed   AcceptsRequestWithChunkedEncodingAndUpgrade
  Error Message:
   Assert.Equal() Failure
                      (pos 9)
  Expected: HTTP/1.1 200 OK
  Actual:   HTTP/1.1 500 In
                      (pos 9)
  Stack Trace:
     at Microsoft.AspNetCore.Testing.TestConnection.<Receive>d__23.MoveNext() in /_/src/Servers/Kestrel/shared/test/TestConnection.cs:line 149
  --- End of stack trace from previous location where exception was thrown ---
     at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
     at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
     at Microsoft.AspNetCore.Server.Kestrel.FunctionalTests.UpgradeTests.<AcceptsRequestWithChunkedEncodingAndUpgrade>d__5.MoveNext() in /_/src/Servers/Kestrel/test/FunctionalTests/UpgradeTests.cs:line 232
  --- End of stack trace from previous location where exception was thrown ---
     at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
     at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
  --- End of stack trace from previous location where exception was thrown ---
     at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
     at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
  --- End of stack trace from previous location where exception was thrown ---
     at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
     at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
```

Not sure of ownership here but @javiercn and @Tratcher you were the last people to change `HttpTests` or `UpgradeTests`&hellip;

Reply:
Does that repro locally? Any chance of getting the logs?

Reply:
Oddly enough, I hit a completely different issue when I tried a release/2.1 build locally: `RequestTests.Request_OptionsStar_EmptyPath()` hits an assertion failure deep in `PathNormalizer.RemoveDotSegments(...)`: `Path segment must always start with a '/'`. That assertion failure when the request URL actually has no path might be worth a look too.

My local build was otherwise completely clean☹️

I'm creating a DTL VM to see how builds work there. Can't exactly match the `1es-windows-2019-open` setup but hopefully `windows.vs2019.amd64.open` is close enough.

Reply:
Which test do you want more info about, the one I deleted here or `AcceptsRequestWithChunkedEncodingAndUpgrade()` (which just failed again in its 3rd attempt).